build/ci/golangci.yml

```diff
@@ -39,6 +39,8 @@ linters-settings:
       - ^.*\.SetBody$
       # Instead of aferocopy.Copy use aferofs.CopyFs2Fs
       - ^aferocopy\.Copy$
+      # Instead of "zap" package use "internal/log" package
+      - ^zap\.
```

Comment:
Commit `Prohibit the use of the zap package in the project`:

-----------

Zakzanie pouzitia `zap` (logger) balicka v projekte.